### DISCUSSION TITLE
Broken: Eager Loading of relationship(s)

### DISCUSSION THREAD

**Discussion by adm-bome** [CodeDredd/pinia-orm#1451] [General]
I could be mistaking here. but why when i do:

```js
const withRelations = useRepo(Model).all() //expeted result: models without relations
const withoutRelations = useRepo(Model).withAll().all()  //expeted result: models with relations
```

After data was saved to the model. Relations are already populated even when starting a new query.
And i can't load realtions, that were never saved via a dataset. the inversed relation of a relation for example.

This is realy a pain, and in my oppinion a issue with a **realy high** fix level.

May also be related to issue #1447, because when not providing data an relation is still executed on saving.

Also mentiond in disscussion #1415

**Comment by CodeDredd** [comment]
Answered in #1455
